### DISCUSSION TITLE
localRecord doesn't take into account mute feature

### DISCUSSION THREAD

**Discussion by al3x88** [HaishinKit/HaishinKit.swift#1440] [Q&A]
If I stream with localRecord option:
`publish(streamName, type: .localRecord)`

the local file created doesn't take into account the "mute" feature, so that the audio is on 100% of the time.

My mute snippet:

```swift
rtmpStream.audioSettings = [
                .muted: true, // if you want a mute audio.
            ]
```

Is this a bug or just me doing something wrong?
Is there any workaroud ?

**Comment by shogo4405** [comment]
I've forgotten the details in the old version. If it can't be done by trying, I don't think it's possible. I also don't think there's any workaround.
In the latest version, I think it's possible with `stream.audioMixerSettings.isMuted = true`.

**Reply by al3x88** [reply]
Thanks!! So in the latest version this issue is not present?
